validator/src/main/java/com/amazon/aoc/MetricContext.java

```diff
@@ -0,0 +1,9 @@
+package com.amazon.aoc;
+
+import lombok.Data;
+
+@Data
+public class MetricContext {
```

Comment:
since this validator aims to support multiple component, not just cloudwatch, maybe giving it a concrete name would be better?

Reply:
Changed to CloudWatchContext

Reply:
Good question. Prometheus metrics are the only one being tested. However these metrics are pushed to CloudWatch using structured log, which explains why we test metrics and logs both.